bump.js

```diff
@@ -0,0 +1,58 @@
+/**
+ * Usage:
+ * Change directory to project folder
+ * node /path/to/script versionNumber
+ */
+const fs = require('fs');
+
+// File Location
+const mdFile = "./README.md";
+const packageFile = "./package.json";
```

Comment:
Let's try creating an array of object literals.  I know that might sound weird coming from Java, but making this a little more data driven will be:

Cleaner & less code
Easier to extend
Easier to maintain

Each object can maybe look something like this `{ path: './README.md', pattern: /\d{1,3}.\d{1,3}.\d{1,3}/, }`